src/api/contentful.api.js

```diff
@@ -0,0 +1,21 @@
+import { ContentfulResponseMapper } from './contenful.mappper'
+import { CONTENTFUL_SPACE_ID, CONTENTFUL_ACCESS_TOKEN } from '../lib/constants'
+
+const contentful = require('contentful')
+
+class PostsApi {
+    constructor() {
+        this.posts = new ContentfulResponseMapper()
```

Comment:
This name is misleading, it should be named to this.mapper

Reply:
You wrote early move this.post to api. Where must be this.post then?